02-create.md

```diff
@@ -418,3 +418,14 @@ but it does find the copy in `thesis` that we didn't delete.
 > The command `ls -t` lists things by time of last change,
 > with most recently changed files or directories first.
 > In what order does `ls -R -t` display things?
+
+> ## Creating Files in a Different Way {.challenge}
+> 
+> In this lesson, you learned how to create a text file using the nano command. Now, in the Unix (i.e. Mac OS/Linux) terminal, try the following command in your directory:
```

Comment:
I suggest: 
- "the nano command" -> "`nano`", or maybe a text editor (some people use other editors), as everybody should be familair with it now.
- ~~"in the Unix (i.e. Mac OS/Linux) terminal"~~, this may be confusing for people using Git Bash and everyone knows now what the terminal is.
- "in your directory", which directory? :)